### DISCUSSION TITLE
Boxed comments

### DISCUSSION THREAD

**Discussion by Alexander-N** [tlaplus/vscode-tlaplus#203] [General]
The TLA+ Toolbox provides to functionality to box comments like this:
```TLA
(***************************************************************************)
(* The algorithm is essentially derived by ensuring that this formula Inv  *)
(* is always true.                                                         *)
(***************************************************************************)
```
Looking through https://github.com/tlaplus/Examples this style of commenting seems to be pretty common, see [this spec](https://github.com/tlaplus/Examples/blob/master/specifications/PaxosHowToWinATuringAward/Paxos.tla) for an example by Leslie Lamport.

I'm using the [Comment Box](https://github.com/SlySherZ/vscode-comment-box) extension to get similar results with these settings:
```json
  "commentBox.styles": {
    "defaultStyle": {
      "commentStartToken": "(**",
      "commentEndToken": "**)",
      "leftEdgeToken": "(* ",
      "rightEdgeToken": " *)",
      "topEdgeToken": "*",
      "bottomEdgeToken": "*",
      "topRightToken": "**)",
      "bottomLeftToken": "(**",
      "capitalize": false,
      "ignoreInnerIndentation": false,
      "ignoreOuterIndentation": false,
      "removeEmptyLines": false,
      "textAlignment": "left"
    }
  }
```
Is this worth commenting on in the wiki?

**Comment by alygin** [comment]
It surely is. Here's a new page in the Wiki: [Boxed Comments](https://github.com/alygin/vscode-tlaplus/wiki/Boxed-Comments). Thanks for bringing this up!
